core/modules/file/file.install

```diff
@@ -1141,3 +1141,24 @@ function file_update_1006() {
     db_drop_table('file_display');
   }
 }
+
+/**
+ * Grant the admin role permission to manage files and file types.
+ */
+function file_update_1007() {
+  $admin_role = config_get('system.core', 'user_admin_role');
+
+  $config = config('user.role.' . $admin_role);
+  $permissions = $config->get('permissions');
```

Comment:
Can we do a safety-check here to ensure the config file exists?
```
if (!$config->isNew()) {
```

Reply:
The admin role _should_ always exist but having a partial config file for a user role would probably cause all sorts of issues.